packages/ember-routing/tests/system/router_test.js

```diff
@@ -261,5 +261,20 @@ moduleFor('Ember Router', class extends AbstractTestCase {
 
     triggerEvent(handlerInfos, false, ['loading']);
   }
-});
 
+  ['@test transitionTo should throw an error when called after owner is destroyed'](assert) {
+    let router = createRouter();
+
+    runDestroy(owner);
+
+    router.currentRouteName = 'route-a';
+
+    assert.throws(function() {
+      router.transitionTo('route-b');
+    }, "A transition was attempted from 'route-a' to 'route-b' but the application instance has already been destroyed.");
```

Comment:
This will work fine once you migrate to `expectAssertion`, but I figured I’d point out that this definitely doesn’t do what you expect with `assert.throws`. Specifically, passing `assert.throws` two arguments (a function and a string) does not use the provided string as a matcher for the error message but instead uses the second arg as the label for the assertion. It’s a super super trolly api :sob:

Reply:
sad face